frontend/src/components/OdhDocCard.tsx

```diff
@@ -138,22 +136,31 @@ const OdhDocCard: React.FC<OdhDocCardProps> = ({ odhDoc, favorite, updateFavorit
       isSelected={selected}
       isSelectable
     >
-      <CardHeader>
-        <CardHeaderMain style={{ maxWidth: '33%', width: '100%' }}>
-          <BrandImage
-            src={odhDoc.spec.img || odhDoc.spec.icon || ''}
-            alt={odhDoc.spec.displayName}
-          />
-        </CardHeaderMain>
-        <CardActions hasNoOffset>
-          <FavoriteButton isFavorite={favorite} onClick={() => updateFavorite(!favorite)} />
-        </CardActions>
+      <CardHeader
+        actions={{
+          actions: (
+            <>
+              <FavoriteButton isFavorite={favorite} onClick={() => updateFavorite(!favorite)} />
+            </>
+          ),
+          hasNoOffset: true,
+          className: undefined,
+        }}
+      >
+        {
+          <>
+            <BrandImage
```

Comment:
Well this doesn't bode well... we have duplicate code 🤔 This is our fault... but same comment as last one. Just render `BrandImage` I imagine.